src/promo-manager/core/controller.ts

```diff
@@ -203,8 +203,6 @@ export class Controller {
         this.stateActions.removeFromQueue(slug);
 
         this.updateProgressInfo(slug);
-
-        this.triggerNextPromo();
```

Comment:
This change dont break/fix any test. Can we add test for this?

Reply:
No, this change breaks 8 tests for callback validation `controller.subscribe(callback)`. They are corrected by emitChange calls [here](https://github.com/gravity-ui/onboarding/pull/133/files#diff-ac67e93936749d285e7647db8d74989b7d3f4965fda4c498704524ac548df6c7R636)

Reply:
Maybe we should make closePromoWithTimeout async and await result?

Reply:
We still need to call `addPromoToFinished` and `removeFromQueue` before await result. It's unnecessary here